test/sync/test_dataHandlers.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
 
   'test set default collisionHandler': function(done) {
     var dataHandlers = dataHandlersModule({defaultHandlers: defaultHandlers});
-    dataHandlers.handleCollision(id, metaData, [],  function(err) {
+    dataHandlers.handleCollision(id, metaData, [], function(err) {
```

Comment:
I should be changed everywhere. That function should invoke the default `handleCollision` function.